src/alephbet.coffee

```diff
@@ -1,9 +1,10 @@
-utils = require('./utils.js.coffee')
-Storage = require('./storage.js.coffee')
-adapters = require('./adapters.js.coffee')
+utils = require('./utils')
+adapters = require('./adapters')
+options = require('./options')
+LocalStorageAdapter = require('./local_storage_adapter')
 
 class AlephBet
-  @options = {debug: false}
+  @options = options
```

Comment:
I think for consistency, we can just use the required `options` instead of assigning it internally (same way as `utils.log`)